### ISSUE TITLE
Default color fails color contrast on web pages with white background

### ISSUE THREAD

**Issue by renatoi** [sarbbottam/focus-indicator#14]
Although it is not a text, I believe the default color contrast should pass WCAG 2.0 AA of 4.5 to 1 on web pages with white background (which I believe it is the most common).

The current color contrast is 1.6 on white backgrounds.

**Comment by sarbbottam** [comment]
User will have choose their preferred color, once #11 is resolved.
